.golangci.yml

```diff
@@ -67,7 +67,6 @@ linters:
     - funcorder # checks the order of functions, methods, and constructors
     - funlen # tool for detection of long functions
     - gocheckcompilerdirectives # validates go compiler directive comments (//go:)
-    - gochecknoglobals # checks that no global variables exist
```

Comment:
@StevenACoffman, I disabled the global check here because the generated code is expected to have global variables. And it's hard to totally avoid the global variables, maybe we could ask to change if the global variable is unnecessary, instead of forcing it in CI.

Reply:
Yeah, good call. This setting was carried over from my work stuff where I avoid globals. The original author of gqlgen had used globals more pervasively and it's hard to undo that at this point.